prime-dotnet-webapi/Services/VerifiableCredentialService.cs

```diff
@@ -232,16 +234,17 @@ private async Task<JObject> IssueCredential(string connectionId, int enrolleeId)
         // Create the credential offer.
         private async Task<JObject> CreateCredentialOfferAsync(string connectionId, JArray attributes)
         {
-            var issuerDid = await _verifiableCredentialClient.GetIssuerDidAsync();
+            var ISSUER_DID = await _verifiableCredentialClient.GetIssuerDidAsync();
```

Comment:
Should probably be `issuerDid` since it isn't a constant when you're pulling it from their API, and similarly for `schemaId`